device/transport/mqtt/devdoc/mqtt_requirements.md

```diff
@@ -348,3 +417,22 @@ interface MethodMessage {
 ```
 **]**
 
+### inputMessage event
+
+**SRS_NODE_DEVICE_MQTT_18_057: [** If there is a listener for the `inputMessage` event, a `inputMessage` event shall be emitted for each message received. **]**
```

Comment:
>If there is a listener for the `inputMessage` event, [](start = 35, length = 52)

you can emit even if there are no listeners, it is not going to crash. not sure if there's value if that first "if" statement since we're using enable/disable anyway?